### PR TITLE
Minor revisions

### PR DESCRIPTION
**Pull request by hneth**
This PR only includes various small improvements (e.g., further revising vignettes).  

Unfortunately, my local edits in `test-mytree.R` happened before merging the last PR, so they may create conflicts now.  Hope that you can resolve them without hassles...

### CODE REVIEW

**Comment by ndphillips** [comment]
Can you resolve merge conflicts on your end first? You just need to pull the latest version from master, then merge into your branch locally. You'll then see the merge conflicts. Fix them and recommit!

**Comment by hneth** [comment]
Ok, seems to have worked now.  (I earlier tried to use the Web Editor to resolve the conflicts, but this didn't work on my browser.)   Feel free to approve & merge, as you see fit...

File: tests/testthat/test-mytree.R [hunk 1]
```diff
@@ -67,19 +66,22 @@ test_that("Can build tree based off of custom tree in words (v2)", {
...
     data.test = heart.test,
     decision.labels = c("Healthy", "Disease"),
     my.tree = "If sex = 1, predict Disease.
-                             If age < 45, predict Healthy.
-                             If thal = {fd, normal}, predict Healthy,
-                             Otherwise, predict Disease."
-  )
+               If age < 45, predict Healthy.
+               If thal = {fd, normal}, predict Healthy,
+               (and ignore the rest of this sentence)."
```
Comment (ndphillips) [inline tests/testthat/test-mytree.R:83]: :)

**Review by ndphillips — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**